### PR TITLE
Add scaling for HPX dataloaders, code cleanup

### PR DESCRIPTION
**Pull request by daviddpruitt**
# Modulus Pull Request

## Description
This adds scaling values for constant values in Healpix datasets.
If scaling values are given they will also be required for constant fields.
Removed duplicate code from coupled healpix dataloader to eliminate differing behaviour.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: modulus/datapipes/healpix/data_modules.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def open_time_series_dataset_classic_prebuilt(
     if not ds_path.exists():
         raise FileNotFoundError(f"Dataset doesn't appear to exist at {ds_path}")
 
-    result = xr.open_zarr(ds_path, chunks={"time": batch_size})
+    result = xr.open_zarr(ds_path)
```
Comment (akshaysubr) [inline modulus/datapipes/healpix/data_modules.py:215]: Removing this chunks argument is reasonable, but what were the specific reasons for that here?

File: modulus/datapipes/healpix/timeseries_dataset.py [hunk 2]
```diff
@@ -189,24 +189,35 @@ def __init__(
...
+        self.constants = None
         if self.scaling:
             self._get_scaling_da()
 
+        # setup constants
+        if "constants" in self.ds.data_vars:
+            # extract from ds:
+            const = self.ds.constants.values
+
+            if self.scaling:
```
Comment (akshaysubr) [inline modulus/datapipes/healpix/timeseries_dataset.py:202]: Not `self.constant_scaling` here?

File: modulus/datapipes/healpix/timeseries_dataset.py [hunk 3]
```diff
@@ -189,24 +189,35 @@ def __init__(
...
+                const = (const - self.constant_scaling["mean"]) / self.constant_scaling[
+                    "std"
+                ]
+
+            # transpose to match new format:
+            # [C, F, H, W] -> [F, C, H, W]
+            self.constants = np.transpose(const, axes=(1, 0, 2, 3))
+
+        self.get_constants()
```
Comment (akshaysubr) [inline modulus/datapipes/healpix/timeseries_dataset.py:211]: Why is this needed here?

**Review by akshaysubr — APPROVED** [review]
Looks good to me overall. Just had a few minor comments.

File: modulus/datapipes/healpix/timeseries_dataset.py (hunk 3 above)
Comment (daviddpruitt) [reply modulus/datapipes/healpix/timeseries_dataset.py:211]: Bottom line isn't needed, it should have been deleted. Fixed.

File: modulus/datapipes/healpix/data_modules.py (hunk 1 above)
Comment (daviddpruitt) [reply modulus/datapipes/healpix/data_modules.py:215]: Not that I could find, I suspect this was hardcoded for a specific dataset.

File: modulus/datapipes/healpix/timeseries_dataset.py (hunk 2 above)
Comment (daviddpruitt) [reply modulus/datapipes/healpix/timeseries_dataset.py]: Should be, fixed.
